west.yml

```diff
@@ -34,7 +34,7 @@ manifest:
   # repositories.
   projects:
     - name: ci-tools
-      revision: cfb83fd6f1f498fa05d788a5d2cbeff9c92742f5
+      revision: pull/133/head
```

Comment:
is this correct?  Pointing to a PR?

Reply:
yes, until https://github.com/zephyrproject-rtos/ci-tools/pull/133 is merged.

Reply:
now pointing to the merged commit in ci-tools